recipes/xqilla/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+required_conan_version = ">=1.40.0"
+
+
+class ConanXqilla(ConanFile):
+    name = "xqilla"
+    
+    description = (
+        "XQilla is an XQuery and XPath 2 library and command line utility written in C++, implemented on top of the Xerces-C∞ library"
+    )
+    topics = ("conan", "xqilla", "XML", "XQuery")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://xqilla.sourceforge.net/HomePage"
+    license = "Apache-2.0"
+    settings = "os", "arch", "compiler", "build_type"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    exports_sources = "patches/**"
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("xerces-c/3.2.3")
+
+    @property
+    def _doc_folder(self):
+        return os.path.join(
+            self._source_subfolder,
+            "doc"
+        )
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("xqilla is just supported for Linux")
```

Comment:
Looks like xqilla supports windows (the 2.3.3 release log mentions improved Windows support.
Did you test with a compiler that targets windows? mingw/Visual Studio?

If the only reason you exclude other platforms is lack of testing, then please change the error message.
```suggestion
            raise ConanInvalidConfiguration("The xqilla recipe currently only support Linux.")
```

Reply:
Changed to suggestion.